### PR TITLE
Parsing brand families links

### PR DESCRIPTION
**Pull request by bavaria95**
#1

### CODE REVIEW

File: data/api/watches/Ace Collections.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (dm-boiko) [inline data/api/watches/Ace Collections.json]: do we need those files ?

**Review by dimaboyko — CHANGES_REQUESTED** [review]

File: data/api/watches/Ace Collections.json (hunk 1 above)
Comment (bavaria95) [reply data/api/watches/Ace Collections.json]: Yes, but better to be stored at different place indeed. Removed from the repo

File: scripts/get_models.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    brands = json.loads(data)['brands']
+
+    API_KEY = os.environ['API_KEY']
+
+    for brand in brands:
+        brand_name = brand['name']
+        brand_id = brand['id']
+        print(brand_name)
+
+        r = requests.post('http://api.watchbase.com/v1/watches', data = {'key': API_KEY,
```
Comment (dm-boiko) [inline scripts/get_models.py]: would be nice not to have that in repo ;)

File: scripts/get_watch.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+    watches_to_download = list(set(watches) - set(downloaded_watches))
+    print("Remains to download: %s" % len(watches_to_download))
+
+    return watches_to_download
+
+
+def process_page(watch):
+    brand, family, model = watch
+    url = 'https://watchbase.com/%s/%s/%s' % (brand, family, model)
```
Comment (dm-boiko) [inline scripts/get_watch.py]: same here, as env var maybe

File: scripts/get_watches.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+    return watches_links
+
+
+def main():
+    families_for_scrape = get_needs_to_scrape()
+
+    for brand, family in families_for_scrape:
+        print(brand, family)
+        url = 'https://watchbase.com/%s/%s' % (brand, family)
```
Comment (dm-boiko) [inline scripts/get_watches.py]: same

File: scripts/watches/middlewares.py [hunk 5]
```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
```
Comment (dm-boiko) [inline scripts/watches/middlewares.py]: do we need all those files ?

**Review by dimaboyko — CHANGES_REQUESTED** [review]

File: scripts/get_models.py (hunk 2 above)
Comment (bavaria95) [reply scripts/get_models.py]: Makes sense

File: scripts/watches/middlewares.py (hunk 5 above)
Comment (bavaria95) [reply scripts/watches/middlewares.py]: Now - not really. If ever we decide to customize the scraper - then maybe. But can be without them now.

File: data/brands_urls.json [hunk 6]
```diff
@@ -0,0 +1,474 @@
+[
```
Comment (dm-boiko) [inline data/brands_urls.json:1]: maybe just ["ulr1", "url2"]

File: scripts/get_watch.py [hunk 7]
```diff
@@ -0,0 +1,104 @@
+import os
```
Comment (dm-boiko) [inline scripts/get_watch.py:1]: I could only change naming, file => according to what is getting called in `main`

**Review by dimaboyko — APPROVED** [review]
good thing - there is data to work with ;)

**Pull request merged**